### ISSUE TITLE
query51a - erroneous line break

### ISSUE THREAD

**Issue by gregrahn** [gregrahn/tpcds-kit#33]
Line 90 has an erronous line break causing the column alias to be on two lines.  Seems like an obvious mistake.

```sql
select v1.item_sk, v1.d_date, v1.web_sales, v1.store_sales, max(v2.web_sales) web_cumulative, max(v2.store_sales) s
tore_cumulative
```

**Comment by gregrahn** [comment]
fixed with 9ef085d

**Comment by gregrahn** [comment]
Present in v2.4.0

**Comment by gregrahn** [comment]
Present in v2.6.0

**Comment by gregrahn** [comment]
Fixed in v2.9.0
